src/custom/pages/App/index.tsx

```diff
@@ -90,16 +91,28 @@ function createRedirectExternal(url: string) {
 
 const Loading = <LoadingWrapper>Loading...</LoadingWrapper>
 
+const getShowBannerState = (key: string) => {
+  const localStorageValue = localStorage.getItem(key)
+
+  // item doesn't exist, show banner (true)
+  if (localStorageValue === null) return true
+
+  // else return localstorage state (!! for type safety)
+  return !!JSON.parse(localStorageValue)
+}
+
 export default function App() {
   // Dealing with empty URL queryParameters
   useFilterEmptyQueryParams()
 
+  const isSideBannerVisible = useMemo(() => getShowBannerState(IS_SIDE_BANNER_VISIBLE_KEY), []) // empty array only calls on mount
```

Comment:
maybe for another day but this shouldnt in here, let's move this to a smart container for the banner itself and no pollute this page

@fairlighteth can you make a note of this?

Reply:
Yes noted, thank you.